.github/workflows/inspecjs-tests.yml

```diff
@@ -41,6 +44,12 @@ jobs:
 
      - name: Run unit tests
        run: yarn inspecjs test
+    
+     - name: Validate there are no changes between the inspec schemas and generated parsers
+       run: |
+        gem install inspec-bin -v 4.38.3
```

Comment:
I know I originally said we wanted to just do a `gem install` here, but the more I think about it I can see a benefit to just adding a `Gemfile` and `Gemfile.lock`, just so that we can get Dependabot updates automatically for testing against new inspec versions. Thoughts?

Reply:
Yeah I completely agree. I was thinking the one of the main points of running the inspecjs tests was to make sure nothing has been altered in the generated parsers of inspec schemas. So it would be nice to also know when something has changed on inspec's end upon a new version instead of just comparing against the schemas of one version of inspec.